src/components/TopPage/TopPage.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useCallback } from 'react'
+import { composeClasses } from 'lib/classes'
+import Breadcrumbs, { BreadcrumbsProps } from 'components/Breadcrumbs'
+import Flex from 'components/Layout/Flex'
+import Circle from 'components/Circle'
+import { Button } from 'components/Buttons'
```

Comment:
```suggestion
import  Button  from 'components/Buttons'
```

Reply:
I have this error when making the change:
Module '"/Users/ariel/bui/src/components/Buttons/index"' has no default export. Did you mean to use 'import { Button } from "/Users/ariel/bui/src/components/Buttons/index"' instead?haz